modules/hal_nordic/Kconfig

```diff
@@ -72,6 +72,8 @@ choice NRF_802154_SL_TYPE
 
 config NRF_802154_SL_OPENSOURCE
 	bool "Open source"
+	select SENSOR
```

Comment:
Should this be `depends on` instead of `select`? We don't usually select driver classes.

Reply:
Wouldn't that just introduce another step for the user to configure the driver?
I my opinion this is a cleaner version, as the temperature driver is required for this feature to work, and the user is not concerned with that fact.

Reply:
Please see here: https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#select-statements